lochness/xnat/__init__.py

```diff
@@ -173,13 +176,24 @@ def download_xnat_session_dataorc(
         subject (str): The ID of the subject.
         session (str): The ID of the session to download.
         out_dir (str): The directory to which the session data should be downloaded.
+        xnat_username (str): The username to use when authenticating with the XNAT server.
+        xnat_password (str): The password to use when authenticating with the XNAT server.
 
     Returns:
         None
     """
 
+    dataorc_binary_path = Path('/opt/software/dataorc-bin/dataorc')
+
+    def clear_stored_credentials(dataorc_bindary_path: Path):
+        cli = f"{dataorc_bindary_path} reset-credentials"
+        subprocess.run(cli, shell=True, timeout=10, stdout=subprocess.DEVNULL)
+
+    logger.info('Clearing stored credentials for dataorc')
+    clear_stored_credentials(dataorc_binary_path)
```

Comment:
I am clearing the credentials before each `dataorc` call, so that `dataorc` asks for the credentials, and them I provide them when prompted.

So for each new XNAT download:
1. Clear `dataorc` credentials, if cached from prior run
2. Run the download `dataorc` cli call
3. Provide credentials.
  i. Since we have cleared the credentials before, we know that this will always happen.
4. Wait for the `dataorc' call to succeed.

Reply:
So will a human need to be on-top of running Lochness now or are you saving the username-password in an (encrypted) file and providing from there?

Reply:
Username and Password will be used from Lochness' keyring, and will be entered by Lochness - no human intervention necessary!